package.json

```diff
@@ -29,10 +29,5 @@
   "scripts": {
     "test": "make test-cov"
   },
-  "licenses": [
-    {
-      "type": "BSD",
-      "url": "http://github.com/hapijs/bassmaster/raw/master/LICENSE"
-    }
-  ]
+  "license": "BSD"
```

Comment:
Heya! I'm the new maintainer of bassmaster.

Could you update this to be "BSD-3-Clause"